### PR TITLE
Reformat code

### PR DESCRIPTION
**Pull request by nikhilnanivadekar**

### CODE REVIEW

File: deck-of-cards-kata/src/main/java/bnymellon/codekatas/deckofcards/custom/collections/MutableCollection.java [hunk 1]
```diff
@@ -21,22 +21,28 @@
...
 
-    default boolean addAllIterable(Iterable<T> iterable) {
-        if (iterable instanceof Collection<T> collection) {
+    default boolean addAllIterable(Iterable<T> iterable)
+    {
+        if (iterable instanceof Collection<T> collection)
+        {
             return this.addAll(collection);
-        } else {
+        } else
```
Comment (donraab) [inline deck-of-cards-kata/src/main/java/bnymellon/codekatas/deckofcards/custom/collections/MutableCollection.java]: This is formatted wrong. The else should be on a separate line.

File: deck-of-cards-kata/src/main/java/bnymellon/codekatas/deckofcards/custom/collections/RichIterable.java [hunk 2]
```diff
@@ -59,8 +59,7 @@
             if (result == null)
             {
                 result = each;
-            }
-            else
+            } else
```
Comment (donraab) [inline deck-of-cards-kata/src/main/java/bnymellon/codekatas/deckofcards/custom/collections/RichIterable.java]: See previous comment on else.

**Review by donraab — APPROVED** [review]
There were two issues I saw with else statements not being properly formatted, but the rest LGTM.

File: deck-of-cards-kata/src/main/java/bnymellon/codekatas/deckofcards/custom/collections/MutableCollection.java (hunk 1 above)
Comment (nikhilnanivadekar) [reply deck-of-cards-kata/src/main/java/bnymellon/codekatas/deckofcards/custom/collections/MutableCollection.java]: fixed

File: deck-of-cards-kata/src/main/java/bnymellon/codekatas/deckofcards/custom/collections/RichIterable.java (hunk 2 above)
Comment (nikhilnanivadekar) [reply deck-of-cards-kata/src/main/java/bnymellon/codekatas/deckofcards/custom/collections/RichIterable.java]: fixed

File: deck-of-cards-kata/src/main/java/bnymellon/codekatas/deckofcards/custom/collections/HashBag.java [hunk 3]
```diff
@@ -72,22 +72,22 @@ public void forEachWithIndex(BiConsumer<? super T, Integer> biConsumer)
...
         Counter index = new Counter();
         this.backingMap.forEach((key, count) ->
-        {
-            for (int i = 0; i < count; i++)
-            {
-                biConsumer.accept(key, index.getCount());
-                index.increment();
-            }
-        });
+                                {
```
Comment (donraab) [inline deck-of-cards-kata/src/main/java/bnymellon/codekatas/deckofcards/custom/collections/HashBag.java:75]: This is a not what I would expect. Thoughts?

Comment (nikhilnanivadekar) [reply deck-of-cards-kata/src/main/java/bnymellon/codekatas/deckofcards/custom/collections/HashBag.java:75]: I will fix it later today.

**Comment by nikhilnanivadekar** [comment]
@donraab I am closing this PR due to the lack of cycles I have at the moment. I do not want to hold up other work. Will create a new PR when I get a chance. Thanks!

**Pull request closed without merging**